File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        nums1.insert(nums1.end(),nums2.begin(),nums2.end());
        sort(nums1.begin(),nums1.end());
        int n=nums1.size();
        if(n%2!=0){
            int index=n/2;
            return nums1[index];
        }else{
            double index=nums1[int(n/2-1)]+nums1[int(n/2)];
            return index/2;
        }
    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int[] res=new int[nums1.length+nums2.length];
        for(int i=0;i<nums1.length;i++){
            res[i]=nums1[i];
        }
        for(int j=0;j<nums2.length;j++){
            res[nums1.length+j]=nums2[j];
        }
        Arrays.sort(res);
        int n=res.length;
        if(n%2!=0){
            int index=n/2;
            return res[index];
        }else{
            double val=res[(int)(n/2-1)]+res[(int)(n/2)];
            return val/2;
        }
    }
}